src/drive/targets/public/components/AppRouter.jsx

```diff
@@ -70,6 +70,8 @@ const AppRouter = ({
               <Redirect from="/" to={`onlyoffice/${data.id}`} />
             )}
           </>
+        ) : (
+          <Redirect from="onlyoffice/*" to="/" />
```

Comment:
Maybe we should add a test?

Reply:
Yes of course, I've added tests to check that the right component has been called depending on the input parameters (route, file, onlyoffice activation). I mock the components to limit problems with the redux store